Leanplum-SDK/Classes/Internal/LPConstants.m

```diff
@@ -237,6 +238,7 @@ - (id)init {
 
 NSString *LP_PATH_DOCUMENTS = @"Leanplum_Resources";
 NSString *LP_PATH_BUNDLE = @"Leanplum_Bundle";
+NSString *LP_SWIZZLING_ENABLED = @"LeanplumSwizzlingEnabled";
```

Comment:
This will be read in from the plist and used to set the value of _enableMethodSwizzling, correct?

Reply:
@milos1290 , yes i would like to understand the use case for enableMethodSwizzling

Reply:
@e7mac  _enableMethodSwizzling is not used anymore, i've removed it in latest PR.